packages/base/src/util/findNodeOwner.js

```diff
@@ -0,0 +1,35 @@
+const findNodeOwner = node => {
+	if (!(node instanceof HTMLElement)) {
+		throw new Error("Argument node should be of type HTMLElement");
+	}
+
+	const ownerTypes = [HTMLHtmlElement, HTMLIFrameElement];
+	let currentShadowRootFlag = true;
+	let currentCustomElementFlag = true;
+
+	while (node) {
+		if (node.toString() === "[object ShadowRoot]") {
```

Comment:
This won't work on IE, not sure how relevant it is for screen readers.
document.querySelector("ui5-button").shadowRoot.toString()
"[object Object]"

Please see if there is another way to check this

Reply:
It works on IE, because it doesn't have shadow root. Just like in Chrome for example for custom elements with no shadow root